components/bl-mapbox/component.json

```diff
@@ -208,8 +208,13 @@
         {
           "id": "data",
           "label": "Data"
+        },
+        {
+          "id": "marker",
```

Comment:
Where I could see Market signature?

Reply:
I think we don't need it, I don't see cases where we use the 'marker'. 'coordinates', 'description' and 'data' enough.